src/performance_profiles.jl

```diff
@@ -152,3 +154,49 @@ function performance_profile(
     kwargs...,
   )
 end
+
+"""
+export_performance_profile(T; solver_names = [], logscale::Bool = true, sampletol=0.0, drawtol=0.0,)
```

Comment:
```suggestion
  export_performance_profile(T; solver_names = [], logscale::Bool = true, sampletol=0.0, drawtol=0.0,)
```
And I don't think this signature match the function